### ISSUE TITLE
Need a video for beginers

### ISSUE THREAD

**Issue by Sturmey66** [docker/docs#12352]
File: [get-started/index.md](https://docs.docker.com/get-started/)

The video on this page assumes a developer knowledge that may not be available to all users who need to run docker for business reasons. The video rambles on for 27 minutes and talks about things that regular IT workers don't need, without touching on what they do need to know to be able to download and run an application from the docker store (or whatever they're calling it.)

Can this be written with a little bit more ELI5 and less like I have 4 years of uni and a masters in programming?

**Comment by craig-osterhout** [comment]
Thanks for sharing the issue. The getting started video has been updated. Please take a look and let us know if any feedback.
